src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAOSpec.scala

```diff
@@ -40,7 +40,7 @@ class GraphWorkspaceDAOSpec extends FlatSpec with Matchers with OrientDbTestFixt
     }
   }
 
-  it should "save a new workspace with a realm" in withDefaultTestDatabase { dataSource =>
+  it should "save a new workspace with a realm" in withEmptyTestDatabase { dataSource =>
```

Comment:
It doesn't depend on existing DB, so this eliminates multiple-workspace conflicts.  I just added another while I'm thinking about it.